src/routes/SignIn.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import Button from '../ui-kit/components/Button';
+import Input from '../ui-kit/components/Input';
+
+const Container = styled.div`
+  padding: 16px;
+  border-bottom: 1px solid #e5e5e5;
+  &:last-child {
+    border-bottom: none;
+  }
+`;
+
+const SocialNetworkList = styled.div`
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-item: center;
+`;
+
+const SocialNetworkLink = styled(Link)`
+  border-radius: 50%;
+  border: solid blue 1px;
+  flex-basis: 32px;
+  height: 32px;
+  margin: 8px;
+`;
+
+const Description = styled.p`
+  text-align: center;
+`;
+
+const Form = styled.form`
+  padding: 20px 0;
+`;
+
+const Label = styled.label`
+  display: block;
+  margin-bottom: 10px;
+`;
+
+const StyledInput = styled(Input)`
+  &&& {
+    width: 100%;
+    &&& input {
+      border-radius: 0;
+    }
+  }
+`;
+
+const SignInButton = styled(Button)`
+  &&& {
+    display: block; //probably need additional type of button with display:block
```

Comment:
I think needn't. We can overwrite the styles as you've done in this case. Please, delete the comment